jetty9/src/main/docker/Dockerfile

```diff
@@ -14,35 +14,28 @@
 
 FROM ${docker.openjdk.image}
 
-# create jetty group and user
-RUN groupadd -r jetty \
- && useradd -r -g jetty jetty
-
 # Create env vars to identify image
 ENV JETTY_VERSION ${jetty9.version}
 ENV GAE_IMAGE_NAME jetty
 ENV GAE_IMAGE_LABEL ${docker.tag.long}
 
 # Create Jetty Home
-ENV JETTY_HOME /usr/local/jetty
+ENV JETTY_HOME ${jetty.home}
 ENV PATH $JETTY_HOME/bin:$PATH
 ENV TMPDIR /tmp/jetty
-ADD jetty-distribution.tar.gz /usr/local/
-RUN mv /usr/local/jetty-distribution-* $JETTY_HOME \
- && chown -R jetty:jetty $JETTY_HOME
+RUN mkdir $TMPDIR
+ADD jetty-distribution.tar.gz $JETTY_HOME/..
```

Comment:
@jmcc0nn3ll is there any way to do some Maven magic to have `jetty-distribution.tar.gz` not have all the files in a subdirectory `jetty-distribution-${jetty9.version}` but rather in the root?